org-code-javabuilder/lib/src/main/java/dev/javabuilder/WebSocketServer.java

```diff
@@ -28,7 +29,8 @@
 @ServerEndpoint("/javabuilder")
 public class WebSocketServer {
   private WebSocketInputAdapter inputAdapter;
-  private WebSocketOutputAdapter outputAdapter;
+  private WebSocketOutputAdapter websocketOutputAdapter;
```

Comment:
Does this need to be an instance variable any more?

Reply:
I think if we leave it as an instance variable we get some reuse over multiple executions, I'm not sure if it's strictly necessary though.